backendv2/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"context"
+	"log"
+	"log/slog"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/urfave/cli/v3"
+	"go.opentelemetry.io/otel"
+
+	dltofunightly "github.com/opentofu/registry-ui/cmd/dl-tofu-nightly"
+	getmodulelicense "github.com/opentofu/registry-ui/cmd/get-module-license"
+	getproviderlicense "github.com/opentofu/registry-ui/cmd/get-provider-license"
+	rebuildglobalindexes "github.com/opentofu/registry-ui/cmd/rebuild-global-indexes"
+	syncmodule "github.com/opentofu/registry-ui/cmd/sync-module"
+	syncprovider "github.com/opentofu/registry-ui/cmd/sync-provider"
+	syncproviders "github.com/opentofu/registry-ui/cmd/sync-providers"
+	syncrepostats "github.com/opentofu/registry-ui/cmd/sync-repo-stats"
+	"github.com/opentofu/registry-ui/pkg/config"
+	"github.com/opentofu/registry-ui/pkg/db"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Setup structured logging first
+	setupLogger()
+
+	backendConfig, err := config.LoadConfig()
```

Comment:
A little bit weird, that if I want to run a command that does not need a config, like `dl-tofu-nightly`, it doesn't allow me to do so.
Did you want to make it this way?